jetty-maven-plugin/pom.xml

```diff
@@ -252,14 +252,18 @@
       <artifactId>jetty-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
+    <!--<dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-home</artifactId>
```

Comment:
@olamy why is this dependency here?
When I comment it out, the plugin still works (and tests properly).

Reply:
I'll just delete it.

Reply:
Because jetty-maven-plugin depends on it (see https://github.com/eclipse/jetty.project/blob/3569f6506618f0a4d95f76b72979cc0548cc6a42/jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/AbstractWebAppMojo.java#L68) there is forked mode using/downloaded the distro.
It can work because you download a SNAPSHOT of jetty-home or use one already here locally but it really depends on it.
Try running: `mvn clean install -pl :jetty-maven-plugin -am` with an empty repository.

Reply:
I can see that `jetty-home` doesn't build before `jetty-maven-plugin`, but I really cannot make it fail, even with a new `<version>`!